core/deliverservice/client.go

```diff
@@ -30,18 +30,19 @@ type broadcastSetup func(blocksprovider.BlocksDeliverer) error
 // If further attempts should be made, it returns:
 // 	- a time duration after which the next attempt would be made, true
 // Else, a zero duration, false
-type retryPolicy func(attemptNum int, elapsedTime time.Duration) (time.Duration, bool)
+type retryPolicy func(attemptNum int, elapsedTime time.Duration, yieldLeadership bool) (time.Duration, bool)
```

Comment:
After making a pass through these changes, I don't think this update is needed. I think we can encapsulate the behavior of static or dynamic leaders within the retry functions without changing the contract throughout the code.

I believe that means this file needs no changes.